packages/babel-plugin-transform-runtime/test/fixtures/runtime-corejs3/aliased-constructors/output.js

```diff
@@ -1,8 +1,8 @@
-var _Map = require("@babel/runtime-corejs3/core-js-stable/map");
+var _Promise = require("@babel/runtime-corejs3/core-js-stable/promise.js");
 
-var _Symbol = require("@babel/runtime-corejs3/core-js-stable/symbol");
+var _Symbol = require("@babel/runtime-corejs3/core-js-stable/symbol.js");
 
-var _Promise = require("@babel/runtime-corejs3/core-js-stable/promise");
+var _Map = require("@babel/runtime-corejs3/core-js-stable/map.js");
```

Comment:
I don't think the extension `.js` should be added since we have specified `package.json#exports`.